src/lib/comms/sol-lwm2m-security.c

```diff
@@ -0,0 +1,470 @@
+/*
+ * This file is part of the Soletta (TM) Project
+ *
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#define SOL_LOG_DOMAIN &_lwm2m_security_domain
+
+#include "sol-log-internal.h"
+#include "sol-util-internal.h"
+#include "sol-coap.h"
+#include "sol-lwm2m-security.h"
+#include "sol-platform.h"
+#include "sol-socket-dtls.h"
+#include "sol-socket.h"
+#include "sol-str-slice.h"
+#include "sol-util-internal.h"
+#include "sol-vector.h"
+
+SOL_LOG_INTERNAL_DECLARE(_lwm2m_security_domain, "lwm2m-security");
+
+#ifdef DTLS
+
+struct sol_socket *sol_coap_server_get_socket(const struct sol_coap_server *server);
```

Comment:
Because it's an internal API, as stated here: https://github.com/solettaproject/soletta/blob/master/src/lib/comms/sol-coap.c#L138.

`sol-oic-security.c` does the same, so I just followed the pattern... if I remove it, I get compiler warnings.